### PR TITLE
Marked files for translation

### PR DESCRIPTION
**Pull request by a-gradina**
Marked following files for translations:

- `/components/FatalError.tsx`
- `/components/gocardless/GoCardlessLink.tsx`
- `/components/LoggedInUser.tsx`
- `/components/modals/GoCardlessExternalMsgModal.tsx`
- `/components/manager/BudgetList.tsx`

### CODE REVIEW

File: packages/desktop-client/src/components/manager/BudgetList.tsx [hunk 1]
```diff
@@ -197,7 +206,7 @@ function FileItem({
   return (
     <View
       onClick={() => _onSelect(file)}
-      title={getFileDescription(file) || ''}
+      title={description ? t(description) : ''}
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/manager/BudgetList.tsx:209]: **💬 suggestion:** ‏could you move `t` to inside `getFileDescription` instead? The translations should be as close to the content as possible.

**Review by MatissJanis — CHANGES_REQUESTED** [review]
Just one change request. Would you also mind looking into fixing the failing CI job (I believe the release notes should be a single line and that will fix it).

File: packages/desktop-client/src/components/manager/BudgetList.tsx (hunk 1 above)
Comment (a-gradina) [reply packages/desktop-client/src/components/manager/BudgetList.tsx]: Sure, done!

**Review by MatissJanis — APPROVED** [review]
Great job!

**Pull request merged**